device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-C32/th3-z9332f-32x100G.config.bcm

```diff
@@ -8,7 +8,7 @@ load_firmware=0x2
 
 ccm_dma_enable=0
 ccmdma_intr_enable=0
-mem_cache_enable=0
+mem_cache_enable=1
```

Comment:
we need sram as well.

Reply:
Hi @lguohan,

Do you mean sram_scan_enable flag?.